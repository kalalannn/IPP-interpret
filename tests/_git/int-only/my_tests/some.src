.ippcode19
defvar GF@first
defvar GF@second
move GF@second int@10
move GF@first GF@second
createframe
defvar TF@new
defvar TF@som
pushframe
