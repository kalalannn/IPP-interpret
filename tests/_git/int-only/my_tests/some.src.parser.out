<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode19">
    <instruction order="1" opcode="DEFVAR">
        <arg1 type="var">GF@first</arg1>
    </instruction>
    <instruction order="2" opcode="DEFVAR">
        <arg1 type="var">GF@second</arg1>
    </instruction>
    <instruction order="3" opcode="MOVE">
        <arg1 type="var">GF@second</arg1>
        <arg2 type="int">10</arg2>
    </instruction>
    <instruction order="4" opcode="MOVE">
        <arg1 type="var">GF@first</arg1>
        <arg2 type="var">GF@second</arg2>
    </instruction>
    <instruction order="5" opcode="CREATEFRAME"/>
    <instruction order="6" opcode="DEFVAR">
        <arg1 type="var">TF@new</arg1>
    </instruction>
    <instruction order="7" opcode="DEFVAR">
        <arg1 type="var">TF@som</arg1>
    </instruction>
    <instruction order="8" opcode="PUSHFRAME"/>
</program>
